### PR TITLE
Latest: SSCS-10112 - Judicial- CCD- Non-standard captialised names are format…

### PR DESCRIPTION
**Pull request by NHAL22**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: test/unit/steps/appointee/appointee-name/Appointee-name.test.js [hunk 1]
```diff
@@ -79,29 +79,57 @@ describe('AppointeeName.js', () => {
...
+            lastName: 'POTTER-Smith'
+          }
+        });
+      });
+    });
+
+
+    describe(`answers() appellant full name with random case`, () => {
+    const NAME = 'Mr,Harry,POTTER';
+      beforeEach(() => {
```
Comment (benouaer) [inline test/unit/steps/appointee/appointee-name/Appointee-name.test.js:117]: formatting looks a bit off here

**Review by hashimalisolirius — APPROVED** [review]

File: test/unit/steps/appointee/appointee-name/Appointee-name.test.js [hunk 2]
```diff
@@ -79,29 +79,57 @@ describe('AppointeeName.js', () => {
...
 
-  [
-    'Mr,HARRY,POTTER',
-    'Mr,harry,potter',
-    'Mr,haRRy,pOttEr',
-    'Mr,harry John,pOttEr'
-  ].forEach(item => {
-    describe(`answers() appellant full name # ${item}`, () => {
+    describe(`answers() appellant full name with hyphen`, () => {
+    const NAME = 'Mr,HARRY,POTTER-Smith';
```
Comment (benouaer) [inline test/unit/steps/appointee/appointee-name/Appointee-name.test.js:83]: formatting looks a bit off, _const_ should probably be inline with beforeEach

File: test/unit/steps/appointee/appointee-name/Appointee-name.test.js [hunk 3]
```diff
@@ -79,29 +79,57 @@ describe('AppointeeName.js', () => {
...
+            firstName: 'HARRY',
+            lastName: 'POTTER-Smith'
+          }
+        });
+      });
+    });
+
+
+    describe(`answers() appellant full name with random case`, () => {
+    const NAME = 'Mr,Harry,POTTER';
```
Comment (benouaer) [inline test/unit/steps/appointee/appointee-name/Appointee-name.test.js:116]: same as above

File: test/unit/steps/identity/appellant-name/AppellantName.test.js [hunk 4]
```diff
@@ -96,43 +96,70 @@ describe('AppellantName.js', () => {
...
 
-  [
-    'Mr,HARRY,POTTER',
-    'Mr,harry,potter',
-    'Mr,haRRy,pOttEr',
-    'Mr,harry John,pOttEr'
-  ].forEach(item => {
-    describe(`answers() appellant full name # ${item}`, () => {
+    describe(`answers() appellant full name with random case`, () => {
+    const NAME = 'Mr,harry John,pOttEr';
```
Comment (benouaer) [inline test/unit/steps/identity/appellant-name/AppellantName.test.js:100]: same as above

File: test/unit/steps/identity/appellant-name/AppellantName.test.js [hunk 5]
```diff
@@ -96,43 +96,70 @@ describe('AppellantName.js', () => {
...
+            firstName: 'harry John',
+            lastName: 'pOttEr'
+          }
+        });
+      });
+    });
+
+
+    describe(`answers() appellant full name with apostrophe`, () => {
+    const NAME = 'Miss,Sarah,O`Brian';
```
Comment (benouaer) [inline test/unit/steps/identity/appellant-name/AppellantName.test.js:133]: same as above

File: test/unit/steps/representative/representative-details/RepresentativeDetails.test.js [hunk 6]
```diff
@@ -67,22 +67,18 @@ describe('RepresentativeDetails.js', () => {
...
-      'mr,harry,potter',
-      'mR,haRRy,pOttEr',
-      'Mr,harry John,pOttEr'
-    ].forEach(item => {
-      it(`should normalise reps full name # ${item}`, () => {
-        representativeDetails.fields.name.title.value = item.split(',')[0];
-        representativeDetails.fields.name.first.value = item.split(',')[1];
-        representativeDetails.fields.name.last.value = item.split(',')[2];
-        expect(representativeDetails.CYAName).to.equal('Mr Harry Potter');
+     beforeEach(() => {
```
Comment (benouaer) [inline test/unit/steps/representative/representative-details/RepresentativeDetails.test.js:76]: you're not doing any setup, so no need for the beforeEach

File: test/unit/steps/representative/representative-details/RepresentativeDetails.test.js [hunk 7]
```diff
@@ -335,16 +331,11 @@ describe('RepresentativeDetails.js', () => {
...
-      'MR,HARRY,POTTER',
-      'mr,harry,potter',
-      'mR,haRRy,pOttEr',
-      'Mr,harry John,pOttEr'
-    ].forEach(item => {
-      it(`should contain a value object with full name # ${item}`, () => {
-        representativeDetails.fields.name.title.value = item.split(',')[0];
-        representativeDetails.fields.name.first.value = item.split(',')[1];
-        representativeDetails.fields.name.last.value = item.split(',')[2];
+  const repName = 'MR,HARRY,POTTER';
```
Comment (benouaer) [inline test/unit/steps/representative/representative-details/RepresentativeDetails.test.js:334]: formatting as above

**Review by benouaer — APPROVED** [review]

File: test/unit/steps/representative/representative-details/RepresentativeDetails.test.js (hunk 6 above)
Comment (NHAL22) [reply test/unit/steps/representative/representative-details/RepresentativeDetails.test.js:76]: i tried it without the beforeEach and it seems to be causing issues on the pipeline

**Pull request merged**